### ISSUE TITLE
FrameNet: provide manual links for high frequency verbs

### ISSUE THREAD

**Issue by LinguList** [clld/concepticon-data#483]
I think, this "framenet" project [here](https://framenet2.icsi.berkeley.edu/fnReports/data/luIndex.xml) may provide some useful links, similar to the resource in #482, which also links to FrameNet. 

What we need to discuss is: if we treat these as metadata, we should probably not link automatically here, but rather have a subset of our date extracted and refine them manually. 

The reason is: automatic links to this kind of metadata will lead to a distortion, as the data is supposed to provide very well-described meanings. We could derive a subset of our concepticon-entries (say: 400 concepts) to make a resource that consistently links to some of these higher-quality semantic resources.

**Comment by xrotwang** [comment]
Yes, I'd even go further and say that combining this kind of data with concepticon will always be very study-specific. So the only reason to explicitly link it from concepticon would be to not lose track of it.

**Comment by LinguList** [comment]
What is also possible is to think of some spin-off project that could be done in the future (if human resources are available), where we refine links for selected high-frequency sub-parts of concepticon, and also correct data and the like. Such a project could also provide automatic links to Wordnet (where possible) or to other lexical resources, and also include a refinement of our definitions. This is far-future, but might be kept in mind to be used in combination with applying for specific funding. For the moment, however, it's not really high-priority, and I would not like this kind of work to be done by an army of Hiwis, as I don't trust students with not enough experience to be able to pursue this. 

So in priority, this comes after our idea to automatically link a couple of metadata resources that can only be automatically linked...
